setup.py

```diff
@@ -1,37 +0,0 @@
-from pathlib import Path
-
-from setuptools import find_packages, setup
-
-parent_dir = Path(__file__).resolve().parent
-
-setup(
-    name="up42-py",
-    version=parent_dir.joinpath("up42/_version.txt").read_text(encoding="utf-8"),
```

Comment:
Our poetry setup uses static version while this one uses external versioning. We either need to use this file in the poetry or drop it completely